tests/tier2/tc_2016_validate_owner_option_in_etc_virtwho_d.py

```diff
@@ -47,7 +48,8 @@ def test_run(self):
         logger.info(">>>step4: owner option is null value")
         self.vw_option_update_value(option_tested, '', config_file)
         data, tty_output, rhsm_output = self.vw_start()
-        msg_list = ["owner not in|owner.* not set|virt-who can't be started|Communication with subscription manager failed"]
+        msg_list = ["owner not in|owner.* not set|virt-who can't be started|"
+                    "Communication with subscription manager failed"]
```

Comment:
how about
```
msg_list = [
    "owner not in
    |owner.* not set
    |Communication with subscription manager failed"]
```

Reply:
Yes, it's better to list each search messages in different lines.
And have updated all msg_list to show like this example in this pr.